aws-amplify/docs
# Delete gen2 banners

#### Description of changes:
- Remove logic from layout for adding gen2 banners
- Delete the `Banners` component folder
  - Remove from `mdx-components` and deleted associated styles

Staging site: https://delete-gen2-banners.d1ywzrxfkb9wgg.amplifyapp.com/

To test:
1. Go to https://delete-gen2-banners.d1ywzrxfkb9wgg.amplifyapp.com/gen1
2. Verify that there are no more gen2 banners being shown

#### Related GitHub issue #, if available:

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?

Which platform(s) are affected by this PR (if applicable)?

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks




      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._
